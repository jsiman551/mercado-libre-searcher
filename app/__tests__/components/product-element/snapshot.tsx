import { render } from '@testing-library/react'
import ProductElement from '@/components/product-element'

export const mockProps = {
  title: 'Casa En Venta En Ecuestre (new)',
  price: 500,
  address: {
    state_name: 'Bs.As. Costa AtlÃ¡ntica',
    city_name: 'Costa Esmeralda',
  },
  thumbnail: 'http://http2.mlstatic.com/D_934244-MLA72205123280_102023-I.jpg',
  shipping: {
    free_shipping: false,
  },
  condition: 'New',
  permalink:
    'https://casa.mercadolibre.com.ar/MLA-1537731218-casa-en-venta-en-ecuestre-_JM',
}

it('renders Product Element Component', () => {
  const { container } = render(<ProductElement {...mockProps} />)
  expect(container).toMatchSnapshot()
})
