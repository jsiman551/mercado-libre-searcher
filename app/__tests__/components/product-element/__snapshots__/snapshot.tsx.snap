// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Product Element Component 1`] = `
<div>
  <div
    class="sc-guDMob iyvFoV"
    data-testid="container"
  >
    <div
      class="sc-dmyDmy ejzjv"
    >
      <div
        class="sc-hLQTFJ iSfHxC"
      >
        <a
          href="https://casa.mercadolibre.com.ar/MLA-1537731218-casa-en-venta-en-ecuestre-_JM"
          target="_blank"
        >
          <img
            alt="Casa En Venta En Ecuestre (new)"
            class="sc-eDLKEg jRGps"
            height="160"
            src="http://http2.mlstatic.com/D_934244-MLA72205123280_102023-I.jpg"
            width="160"
          />
        </a>
      </div>
      <div
        class="sc-jTQDJr dmUHMV"
      >
        <p
          class="sc-gLLuHO brhnQY"
        >
          $ 500
        </p>
        <h2
          class="sc-iBdoyZ kjSFWv"
        >
          Casa En Venta En Ecuestre (new) (New)
        </h2>
        <p
          class="sc-fsYfxw gPyZgP"
        >
          En 3 cuotas de $15.333
        </p>
      </div>
    </div>
    <p
      class="sc-qZruQ jObgFK"
    >
      Costa Esmeralda, Bs.As. Costa Atl√°ntica
    </p>
  </div>
</div>
`;
